src/time.cr

```diff
@@ -310,43 +611,41 @@ struct Time
     Time.new(seconds: seconds, nanoseconds: nanoseconds.to_i, location: location)
   end
 
-  # Returns the amount of time between *other* and `self`.
+  # Returns a `Time::Span` amounting to the duration between *other* and `self`.
   #
-  # The amount can be negative if `self` is a `Time` that happens before *other*.
+  # The time span is negative if `self` is before *other*.
+  #
+  # The duration amounts to the actual time elapsed between both instances, on
+  # the instant time-line.
+  # The difference between local date-time representations may equal to a
+  # differnt duration, depending on time zone transitions.
```

Comment:
And leap seconds.

Reply:
`differnt` => `different`

Reply:
Which are ignored - or, more exactly, are not even recognized - so they shouldn't have any influence.

Reply:
To be more clear: Neither the instance time-line nor the date-time representations - which are both models for `Time` - take leap seconds into account. Thus leap seconds have no impact on the difference between both.

Leaps seconds can only lead to a difference of one second to the *actual* time elapsed.